crates/aptos/src/test/mod.rs

```diff
@@ -46,10 +47,28 @@ use serde_json::Value;
 use std::collections::BTreeMap;
 use std::path::PathBuf;
 use std::{str::FromStr, time::Duration};
+use thiserror::private::PathAsDisplay;
 use tokio::time::{sleep, Instant};
 
 pub const INVALID_ACCOUNT: &str = "0xDEADBEEFCAFEBABE";
 
+pub const FIRST_MOVE_FILE: &str = "
```

Comment:
maybe include_str?

Reply:
also this isn't used in this PR

Reply:
Yeah, it's no longer used but I believe I removed it unless you're commenting on a specific commit